addon/styles/_list-item-primitives.scss

```diff
@@ -0,0 +1,185 @@
+@import 'node_modules/ember-frost-core/addon/styles/frost-theme';
+
+$frost-list-item-primary-state-critical: #E52C2C;
+$frost-list-item-primary-state-major: #EF8E00;
+$frost-list-item-primary-state-warn: #7D85B2;
+
+.frost-list-item-primary-identifier {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: $frost-font-s;
+  color: $frost-color-grey-1;
+  font-weight: 600;
+  line-height: 30px;
+}
+
+.frost-list-item-secondary-identifier {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: $frost-font-xs;
+  color: $frost-color-grey-1;
+  line-height: 30px;
+}
+
+.frost-list-item-unlabeled-property {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  font-size: $frost-font-xs;
+  color: $frost-color-grey-1;
+  line-height: 30px;
+}
+
+.frost-list-item-boolean {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 30px;
+  .frost-list-item-boolean-label {
```

Comment:
No need to nest if you're using an full name and not using `&-` (we should probably discuss the convention there before we proceed)